src/elements/content-sidebar/activity-feed/comment-form/CommentForm.scss

```diff
@@ -11,6 +11,7 @@
                     min-height: 68px;
                     max-height: 140px;
                     overflow: auto;
+                    background-color: $white;
```

Comment:
This will affect the existing experience in Preview. Is that desirable?

Reply:
Without this the input background has the same color as the selected/hovered item. Is there a place where we want this behaviour?